src/main/java/com/suse/salt/netapi/results/ResultInfo.java

```diff
@@ -34,7 +34,7 @@
     private String user;
 
     @SerializedName("Target")
-    private String target;
+    private Object target;
```

Comment:
Why does it need to be `Object` here, isn't a compound matcher also a `String` in the end (`Target<String>`)?